src/Display.cpp

```diff
@@ -24,6 +26,10 @@ Display::Display(int width, int height, const std::string& title)
         std::cerr << "An error occured while initializing Glew." << std::endl;
     }
     isClosed_ = false;
+
+	glEnable(GL_DEPTH_TEST);
+	glEnable(GL_CULL_FACE);
+	glCullFace(GL_BACK);
```

Comment:
Make sure you're using spaces, you can probably set that in Visual Studio to use spaces instead of tabs. Looks wonky here with a mix of spaces and tabs.